### DISCUSSION TITLE
Doc reorganization

### DISCUSSION THREAD

**Discussion by jhenin** [Colvars/colvars#688] [Ideas]
I think the main use case of our doc is online in HTML form. The single HTML page is becoming large and unwieldy. I've experimented with tex4ht to generate separate pages with a side frame containing a TOC. This roughly works, but not very well because the Latex code + building toolchain is complex and somewhat fragile.

I'd like to know if there is demand for PDF docs. I suspect we could just drop them and no one would bat an eye. This would remove the need for supporting different cases with latex macros, and let us focus on the main stake, the HTML.

**Comment by jhenin** [comment]
An additional concern in this respect: RAG models are very useful to extract relevant information from a lengthy documentation. However they can only work with limited context sizes. I have seen hallucinations in RAG models trying to answer Colvars questions, that I suspect were due to the doc not fitting in the model context.

**Comment by jhenin** [comment]
Replying to my initial question: apparently there is demand for the PDF doc. Some users download it and use it locally. So any change in the doc system should preserve a PDF output.

**Comment by jhenin** [comment]
Working on a multi-page doc in a readthedocs style...
<img width="1598" height="925" alt="image" src="https://github.com/user-attachments/assets/dd152ec0-ad73-49b8-871b-6bb7b6cbf4f0" />
The main issue I see is that the section/subsection titles are quite long, making the TOC a little harder to read. Working well otherwise.

**Reply by giacomofiorin** [reply]
> The main issue I see is that the section/subsection titles are quite long, making the TOC a little harder to read. Working well otherwise.

Does it work also with removing the section numbers to simplify them? Last time I tried the hyperlinks stopped working, but they might in this new layout...
